tests/pkg_fatfs_vfs/main.c

```diff
@@ -295,6 +295,58 @@ static void test_create(void)
     print_test_result("test_create__umount", vfs_umount(&_test_vfs_mount) == 0);
 }
 
+#ifdef MODULE_NEWLIB
+static void test_newlib(void)
+{
+    FILE* fl;
+    char buf[sizeof(test_txt) + sizeof(test_txt2)];
+    print_test_result("test_newlib__mount", vfs_mount(&_test_vfs_mount) == 0);
+
+    /* try to open file that doesn't exist */
+    fl = fopen(FULL_FNAME_NXIST, "r");
+    print_test_result("test_newlib__fopen", fl == NULL);
+
```

Comment:
`cppcheck` complains about a resource leak, so you should probably add 

```C
if (fl) {
    fclose(fl);
}
```

just in case

Reply:
Please squash!